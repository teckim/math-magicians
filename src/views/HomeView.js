import Card from '@mui/material/Card';
import CardContent from '@mui/material/CardContent';
import Container from '@mui/material/Container';
import Typography from '@mui/material/Typography';

const HomeView = () => (
  <section>
    <Container sx={{ mt: '48px' }}>
      <Card>
        <CardContent>
          <Typography
            gutterBottom
            variant="h5"
          >
            Welcome to our page!
          </Typography>
          <Typography
            paragraph
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean nec lorem
            eget leo faucibus fermentum. Etiam vel lacus lorem. Sed aliquet, nisi quis
            tincidunt pretium, odio ex congue mauris, quis tristique justo ipsum sed
            urna. Aliquam suscipit, purus eu faucibus gravida, est metus egestas quam,
            id feugiat elit ante vel ante. Aliquam a erat vitae tortor scelerisque
            venenatis. Sed tristique orci nisl. Nam rutrum magna et massa hendrerit
            sagittis. Nam luctus odio sit amet eros semper volutpat. Etiam a posuere
            nunc.
          </Typography>
        </CardContent>
      </Card>
    </Container>
  </section>
);

export default HomeView;
