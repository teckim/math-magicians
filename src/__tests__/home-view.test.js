import React from 'react';
import renderer from 'react-test-renderer';
import HomeView from '../views/HomeView';

describe('home view', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(<HomeView />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
