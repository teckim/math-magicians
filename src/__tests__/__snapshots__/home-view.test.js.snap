// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home view should render correctly 1`] = `
<section
  className="container"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean nec lorem eget leo faucibus fermentum. Etiam vel lacus lorem. Sed aliquet, nisi quis tincidunt pretium, odio ex congue mauris, quis tristique justo ipsum sed urna. Aliquam suscipit, purus eu faucibus gravida, est metus egestas quam, id feugiat elit ante vel ante. Aliquam a erat vitae tortor scelerisque venenatis. Sed tristique orci nisl. Nam rutrum magna et massa hendrerit sagittis. Nam luctus odio sit amet eros semper volutpat. Etiam a posuere nunc. Nunc metus eros, egestas sit amet quam et, pharetra posuere risus. Cras suscipit enim quis ante lobortis, eu tempor libero venenatis. Fusce malesuada in tellus ut tincidunt. Pellentesque sed enim at purus fringilla facilisis sed quis tortor. Vestibulum vitae sapien rhoncus, vulputate mauris sit amet, cursus mauris. Fusce molestie dolor non blandit consectetur. Aliquam erat volutpat. Sed viverra in purus vel cursus. Mauris cursus ipsum nec mi pellentesque varius. Nullam facilisis lobortis eros, eu eleifend neque ullamcorper in.
  </p>
</section>
`;
